File: 프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.py
def solution(n):
    # 정수를 문자열로 변환하고, 내림차순으로 정렬한 후 리스트를 문자열로 합침
    sorted_str = ''.join(sorted(str(n), reverse=True))
    # 정수형으로 변환하여 반환
    return int(sorted_str)

File: 프로그래머스/1/12933. 정수 내림차순으로 배치하기/정수 내림차순으로 배치하기.java
import java.util.Arrays;

class Solution {
    public long solution(long n) {
       long answer = 0;  

        String str = Long.toString(n); 
        
 
        char[] charArray = str.toCharArray();
        

        Arrays.sort(charArray);
        

        StringBuilder sb = new StringBuilder(new String(charArray));
        

        sb.reverse();
        
        
        answer = Long.parseLong(sb.toString());
        
        return answer; 
    }
    
}